frontend/css/brainzplayer.less

```diff
@@ -373,6 +373,12 @@
     margin-top: 0;
   }
 
+  .queue-list > ul, li {
```

Comment:
`li` here targets all list item elements. Is that what you were aiming for, or are you looking for `.queue-list > ul, .queue-list > li`?

Reply:
Good catch! Yes, I meant `.queue-list > ul, .queue-list > li`.